src/main/java/it/gov/pagopa/debtposition/controller/pd/crud/api/v1/DebtPositionController.java

```diff
@@ -391,17 +392,23 @@ public ResponseEntity<PaymentPositionModelBaseResponse> getDebtPositionByIUV(
   }
 
   @Override
-  public ResponseEntity<String> updateTransferIbanMassive(
+  public ResponseEntity<UpdateTransferIbanMassiveResponse> updateTransferIbanMassive(
       String organizationFiscalCode,
       String oldIban,
+      int limit,
       UpdateTransferIbanMassiveModel updateTransferIbanMassiveModel) {
 
     int numberOfUpdates =
         paymentPositionService.updateTransferIbanMassive(
```

Comment:
The methods should be moved from `PaymentsService` to `PaymentPositionCRUDService`, because this i the responsibility map:
-  `PaymentsService` = the service for payment workflow (SOAP request from Nodo, internal request from flow reporting)
- `PaymentPositionCRUDService` = All changes in debit position from PT for CRUD to debt position on which they are authorized (segregation codes matter)